src/Illuminate/Collections/Arr.php

```diff
@@ -597,6 +597,25 @@ public static function random($array, $number = null, $preserveKeys = false)
         return $results;
     }
 
+    /**
+     * Select and "pick" columns from a bi-dimensional array.
+     *
+     * @param  array  $array
+     * @param  array  $columns
+     * @return array
+     */
+    public static function select(&$array, $columns)
+    {
+        if (is_string($columns)) {
```

Comment:
I think You can use 
```
$columns = (array) $columns;
```

instead of two If conditions.

Reply:
> I think You can use
> 
> ```
> $columns = (array) $columns;
> ```
> 
> instead of two If conditions.

Thank you for the feedback.
Yes i think that your suggestion is great.
```
>>> (array) "some"
=> [
     "some",
   ]
>>> (array) null
=> []

```